src/components/tile/_tile.scss

```diff
@@ -312,7 +312,9 @@
   }
 
   .#{$prefix}--tile-input {
-    display: none;
+    @include hidden;
+    opacity: 0;
```

Comment:
Just 🤔, do we still think we need additional CSS props...?

Reply:
I think the input field would still be visible if the positioning is off, which is why I left `opacity: 0` in